pwiz_tools/Skyline/Model/SrmDocument.cs

```diff
@@ -1489,6 +1489,13 @@ private SrmDocument MergeMatchingPeptidesUserInfo(IList<PeptideGroupDocNode> pep
         /// <summary>
         /// Return a mass list import if the progress monitor is not cancelled and we are able to read the document
         /// </summary>
+        /// <param name="inputs"></param>

```

Comment:
It's a little weird to leave these empty - we should at least put a half-assed description in there